packages/components/details/details.jsdom.test.mjs

```diff
@@ -0,0 +1,166 @@
+            <li>appointment letters</li>
+            <li>your NHS medical card</li>
+          </ul>
+          <p>Ask your GP practice for help if you can't find your NHS number.</p>
+        </div>
+      </details>
+    `
+
+    details = document.querySelector('.nhsuk-details')
+    summary = document.querySelector('.nhsuk-details__summary')
+    content = document.querySelector('.nhsuk-details__text')
+
+    jest.spyOn(summary, 'addEventListener')
+  })
+
+  describe('Exports', () => {
+    it('should export init function', () => {
+      expect(initDetails).toBeInstanceOf(Function)
+    })
+  })
+
+  describe('Modern browsers', () => {
+    describe('Initialisation', () => {
+      it('should not add event listeners', () => {
+        initDetails()
+
+        expect(summary.addEventListener).not.toHaveBeenCalledWith(
+          'click',
+          expect.any(Function)
+        )
+
+        expect(summary.addEventListener).not.toHaveBeenCalledWith(
+          'keydown',
+          expect.any(Function)
+        )
+      })
+
+      it('should not throw with empty body', () => {
+        document.body.innerHTML = ''
+        expect(() => initDetails()).not.toThrow()
+      })
+
+      it('should not throw with empty scope', () => {
+        const scope = document.createElement('div')
+        expect(() => initDetails({ scope })).not.toThrow()
+      })
+    })
+
+    describe('Expanded content', () => {
+      beforeEach(() => {
+        initDetails()
+      })
+
+      it('should not add attributes', () => {
+        expect(summary).not.toHaveAttribute('role')
+        expect(summary).not.toHaveAttribute('aria-controls')
+        expect(summary).not.toHaveAttribute('aria-expanded')
+        expect(content).not.toHaveAttribute('aria-hidden')
+        expect(content).not.toHaveAttribute('id')
+      })
+    })
+  })
+
+  describe('Older browsers', () => {
+    beforeEach(() => {
+      // Identify HTMLDetailsElement (modern browsers) as HTMLDivElement
+      Object.setPrototypeOf(details, document.createElement('div'))
+    })
+
+    describe('Initialisation', () => {
+      it('should add event listeners', () => {
+        initDetails()
+
+        expect(summary.addEventListener).toHaveBeenCalledWith(
+          'click',
+          expect.any(Function)
+        )
+
+        expect(summary.addEventListener).toHaveBeenCalledWith(
+          'keydown',
+          expect.any(Function)
+        )
+      })
+
+      it('should not throw with empty body', () => {
+        document.body.innerHTML = ''
+        expect(() => initDetails()).not.toThrow()
+      })
+
+      it('should not throw with empty scope', () => {
+        const scope = document.createElement('div')
+        expect(() => initDetails({ scope })).not.toThrow()
+      })
+    })
+
+    describe('Accessibility', () => {
+      beforeEach(() => {
+        initDetails()
+      })
+
+      it('should add to summary the button role', async () => {
```

Comment:
_extremely_ minor but do you think a slightly different sentence structure would read better here, e.g.

- 'should set the summary's role to "button"'
- 'should set the summary's aria-expanded to false'